src/main/scala/com/typesafe/sbt/license/TargetLanguage.scala

```diff
@@ -77,3 +77,19 @@ case object Html extends TargetLanguage {
 
   def htmlEncode(s: String) = org.apache.commons.lang3.StringEscapeUtils.escapeHtml4(s)
 }
+
+case object Csv extends TargetLanguage {
+  val ext = "csv"
+  def documentStart(title: String, reportStyleRules: Option[String]): String = ""
+  def documentEnd(): String = ""
+  def createHyperLink(link: String, content: String): String = s"$content"
```

Comment:
Looking at the sample output that you uploaded, I would say that I am pretty convinced that we should go for the `s"$content ($link)"` option. The "License" section seems free-form enough that tools would have a hard time doing anything other than comparing two of them to see if they are equal. Rather, the (non-link) `category` column seems to be the one that would be of interest to tools.